dnf/base.py

```diff
@@ -217,9 +217,14 @@ def _setup_excludes_includes(self, only_main=False):
                 self.sack.add_excludes(query)
 
     def _setup_modules(self):
-        module_defaults_prioritizer = Modulemd.Prioritizer()
-        # HACK: https://github.com/fedora-modularity/libmodulemd/issues/42
-        module_defaults_prioritizer.add([], 0)
+        try:
```

Comment:
Could you remove this commit from PR?
We decided not to use it because DNF shouldn't silently ignore module metadata if something breaks.